x/exchange/spec/03_messages.md

```
@@ -318,11 +341,11 @@
 
 #### MsgMarketSetOrderExternalIDRequest
 
-+++ https://github.com/provenance-io/provenance/blob/v1.20.0/proto/provenance/exchange/v1/tx.proto#L320-L334
++++ https://github.com/provenance-io/provenance/blob/v1.20.0/proto/provenance/exchange/v1/tx.proto#L349-L363
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)